src/southbridge/intel/common/spi.c

```diff
@@ -1110,29 +1110,76 @@ __weak void intel_southbridge_override_spi(struct intel_swseq_spi_config *spi_co
 {
 }
 
+#if CONFIG_SOUTHBRIDGE_INTEL_COMMON_SPI_SILVERMONT
+#define ILB_BASE_ADDRESS		0xfed08000
+#define SPI_BASE_ADDRESS		0xfed01000
+#define SCS				0xf8
+# define SMIWPEN			(0x1 << 7)
+# define SMIWPST			(0x1 << 6)
+
+#define BCS				0x6c
+# define BCS_SMIWPST		(0x1 << 0)
+# define BCS_SMIWPEN		(0x1 << 1)
+
+#define BCR				0xfc
+# define EISS				(0x1 << 5)
+# define BCR_LE				(0x1 << 1)
+# define BCR_WPD			(0x1 << 0)
```

Comment:
These are redundant, you have the same definitions below for BIOS_CNTL

All you need to do is:

```
#define BIOS_CNTL		(CONFIG(SOUTHBRIDGE_INTEL_COMMON_SPI_SILVERMONT) ? 0xfc : 0xdc)
```

and reuse the same definitions.